lib/node_modules/@stdlib/blas/base/ddot/src/addon.cpp

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
You can remove this file, as I need to revisit and refactor due to deprecated V8 APIs. :(